packages/client-search/src/types/Settings.ts

```diff
@@ -273,6 +273,11 @@ export type Settings = {
    * Overrides Algolia's default normalization.
    */
   readonly customNormalization?: Readonly<Record<string, Readonly<Record<string, string>>>>;
+  
+  /**
+   * Enable personalization for queries by default
+   */
+  readonly enablePersonalization?: boolean;
```

Comment:
In addition, we need to update the CTS: https://github.com/algolia/algoliasearch-client-specs-internal/pull/117/files.